operator-framework/src/test/java/io/javaoperatorsdk/operator/ControllerExecutionIT.java

```diff
@@ -38,7 +37,7 @@ public void configMapGetsCreatedForTestCustomResource() {
         integrationTestSupport.teardownIfSuccess(() -> {
             TestCustomResource resource = testCustomResource();
 
-            integrationTestSupport.getCrOperations().inNamespace(TEST_NAMESPACE).create(resource);
```

Comment:
This files consist undesired changes related to auto refactoring code using IDEA, but it looks good.